app/serializers/board_serializer.rb

```diff
@@ -3,8 +3,30 @@
 class BoardSerializer < ActiveModel::Serializer
   USERS_LIMIT = 4
 
-  attributes :id, :title, :slug, :created_at, :users_count, :private, :boards_count
+  # rubocop:disable Layout/LineLength
+  attributes :id, :title, :slug, :created_at, :users_count, :private, :boards_count, :column_emojis, :next_board_slug, :previous_board_slug
+  # rubocop:enable Layout/LineLength
   attribute :users, if: :include_users?
+  attribute :previous_action_items
+  has_many :action_items
+
+  def previous_action_items
+    return [] unless object.previous_board
+
+    object.previous_board.action_items
+  end
+
+  def previous_board_slug
+    return nil unless object.previous_board
```

Comment:
you don't need this, you are using safe operator, so it will return nil if object doesn't exist